content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar aos serviços em execução no cluster Kubernetes.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Acessando serviços em execução no cluster
+
+No Kubernetes, [nós](/pt-br/docs/concepts/architecture/nodes/), [Pods](/docs/concepts/workloads/pods/) e [serviços](/docs/concepts/services-networking/service/) têm todos seus próprios IPs. Em muitos casos, os IPs dos nós, dos Pods e alguns dos IPs de serviço em um cluster não serão
+roteáveis, portanto, não estarão acessíveis a partir de uma máquina fora do cluster, como seu computador.
+
+### Maneiras de se conectar
+
+Você tem várias opções para se conectar a nós, pods e serviços de fora do cluster:
+
+  - Acesse serviços através de IPs públicos.
+    - Use um serviço com tipo `NodePort` ou `LoadBalancer` para tornar o serviço acessível fora do cluster. Consulte a documentação de [serviços](/docs/concepts/services-networking/service/) e
+[kubectl expose](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+    - Dependendo do ambiente do cluster, isso pode expor o serviço apenas para a rede corporativa, ou pode expô-lo para a Internet. Pense se o serviço que está sendo exposto é seguro. Ele faz sua própria autenticação?
+    - Coloque Pods atrás de serviços. Para acessar um Pod específico de um conjunto de réplicas, como para depurar, coloque um rótulo exclusivo no Pod e crie um novo serviço que selecione este rótulo.
+    - Na maioria dos casos, não deve ser necessário para o desenvolvedor de aplicativos acessar diretamente nós através de seus endereços IP.
+  - Acesse serviços, nós ou Pods usando o Verbo Proxy.
+    - Faz autenticação e autorização do servidor de API antes de acessar o serviço remoto. Use isto se os serviços não forem seguros o suficiente para expor à Internet, ou para obter acesso a portas no IP do nó, ou para depuração.
+    - Proxies podem causar problemas para algumas aplicações web.
+    - Só funciona para HTTP/HTTPS.
+    - Descrito [aqui](#manually-constructing-apiserver-proxy-urls).
+  - Acesse a partir de um nó ou Pod no cluster.
+    - Execute um Pod e, em seguida, conecte-se a um shell nele usando [kubectl exec](/docs/reference/generated/kubectl/kubectl-commands/#exec). Conecte-se a outros nós, Pods e serviços a partir desse shell.
+    - Alguns clusters podem permitir que você faça ssh para um nó no cluster. De lá, você pode conseguir acessar os serviços do cluster. Este é um método não padrão e funcionará em alguns clusters, mas não em outros. Navegadores e outras ferramentas podem ou não estar instalados. O DNS do cluster pode não funcionar.
```

Comment:
Sounds better to me "Este é um método que não é padrão" instead of "Este é um método não padrão".